src/main/java/seedu/address/model/Inventory.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model;
+
+import seedu.address.ui.View;
+
+/**
+ * Encapsulates a possible Inventory entry, either an Item or a Recipe.
+ */
+public abstract class Inventory {
```

Comment:
deceptive naming
try to remove if not too necessary, is an unusual class to have